resources/profiles/Anycubic/machine/Anycubic Kobra 2 0.4 nozzle.json

```diff
@@ -0,0 +1,114 @@
+{
+	"type": "machine",
+	"setting_id": "GM001",
+	"name": "Anycubic Kobra 2 0.4 nozzle",
+	"from": "system",
+	"instantiation": "true",
+	"inherits": "fdm_machine_common",
+	"printer_model": "Anycubic Kobra 2",
+	"default_print_profile": "0.20mm Standard @Anycubic Kobra2",
+	"nozzle_diameter": [
+		"0.4"
+	],
+	"printable_area": [
+		"0x0",
+		"220x0",
+		"220x220",
+		"0x220"
+	],
+	"printable_height": "250",
+	"nozzle_type": "undefined",
+	"auxiliary_fan": "0",
+	"machine_max_acceleration_extruding": [
+		"2500",
+		"2500"
+	],
+	"machine_max_acceleration_retracting": [
+		"2500",
+		"2500"
+	],
+	"machine_max_acceleration_travel": [
+		"3000",
+		"1250"
+	],
+	"machine_max_acceleration_x": [
+		"2500",
+		"2500"
+	],
+	"machine_max_acceleration_y": [
+		"2500",
+		"2500"
+	],
+	"machine_max_acceleration_z": [
+		"800",
+		"800"
+	],
+	"machine_max_speed_e": [
+		"80",
+		"80"
+	],
+	"machine_max_speed_x": [
+		"300",
+		"300"
+	],
+	"machine_max_speed_y": [
+		"250",
+		"250"
+	],
+	"machine_max_speed_z": [
+		"8",
+		"8"
+	],
+	"machine_max_jerk_e": [
+		"10",
+		"10"
+	],
+	"machine_max_jerk_x": [
+		"15",
+		"15"
+	],
+	"machine_max_jerk_y": [
+		"10",
+		"10"
+	],
+	"machine_max_jerk_z": [
+		"2",
+		"2"
+	],
+	"max_layer_height": [
```

Comment:
better to change max_layer_height to 0.32 since you created 0.3mm process for it.

Reply:
Done